plugins/chat/app/controllers/chat/api/summaries_controller.rb

```diff
@@ -10,9 +10,11 @@ def get_summary
     channel = Chat::Channel.find(params[:channel_id])
     guardian.ensure_can_join_chat_channel!(channel)
 
-    strategy = Summarization::Base.selected_strategy
+    strategy = DiscourseAi::Summarization::Models::Base.selected_strategy
```

Comment:
Chat summaries now depend on a plugin that it's not always installed.

Reply:
Indeed, chat must follow the same idea of having it's AI summarization bits extracted and moved into the plugin or simply deleted.